assets/scss/pages/page.scss

```diff
@@ -11,154 +11,12 @@
 }
 
 .p-common__title {
```

Comment:
In this case I want you to leave it, because I've already worked on a typography. But in general, instead of creating `title` class inside `pages` - which doesn't make much sense, consider creating an object like `o-title` that could be reused in multiple places. It could have modifiers like `o-title--big` etc.

Try to think about a given part not through lenses of its location, but more like "what it does and how it could be used in different way".  I think in 90% of situations, when you want to put some class into `common` or `pages` objects ar your answers, especially if something if highly common :)